packages/app-bridge/src/registries/commands/CommandRegistry.ts

```diff
@@ -24,4 +25,5 @@ export type CommandRegistry = CommandNameValidator<{
     openNewPublication: OpenNewPublicationPayload;
     openSearchDialog: OpenSearchDialog;
     closeSearchDialog: CloseSearchDialog;
+    navigateToDocumentSection: NavigateToDocumentSection;
```

Comment:
Wouldn’t this:

```suggestion
    navigateToDocumentPageSection: NavigateToDocumentPageSection;
```

or even this:

```suggestion
    navigateToPageSection: NavigateToPageSection;
```

be more accurate? As Documents per se don’t have sections. It’s always the pages that have sections. I would prefer the latter. Wdyt?

Reply:
Ok, then it’s fine. With the domain model we’re currently working on and the AppBridgeTheme refactoring we’ll change the naming anyways. So let’s keep it for now.